servers/quarkus-server/src/main/proto/table.proto

```diff
@@ -32,24 +32,29 @@ message PContents {
 
 message PIcebergTable {
   string metadata_location = 1;
+  int32 entityType = 2;
```

Comment:
I think the entityType should be first but didn't want to break the proto w/o checking w/ the team